chibuezeayogu/React-Redux
# #161607739 Prevent deletion of author with course

**What does the PR do?**
- Prevent Users from deleting an author that has a course

**Description of Task to be completed?**
-  users should not be able to delete author that has a course
 
**How should this be manually tested?**
- Clone the repository [here](https://github.com/chibuezeayogu/React-Redux.git)
-  cd into `React-Redux` and checkout to this branch `ft-prevent-deletion-of-author-with-course-161607739`
- run the command `yarn dev` to start the application locally.
- run the command `yarn test` to ensure all tests are passing.

**What are the relevant pivotal tracker stories?**
- [#161607739](https://www.pivotaltracker.com/story/show/161607739)

**Screenshots**
- N/A
